deps/wazuh_testing/wazuh_testing/tools/wazuh_manager.py

```diff
@@ -0,0 +1,30 @@
+import os
+import subprocess
+import subprocess as sb
+from wazuh_testing.tools import WAZUH_PATH
+from wazuh_testing.wazuh_db import query_wdb
+
+
+def list_agents_ids():
+    wazuhdb_result = query_wdb("global get-all-agents last_id -1")
+    list_agents = [agent['id'] for agent in wazuhdb_result if not (0 == agent.get('id'))]
+
+    return list_agents
+
+
+def remove_agents(agents_id, remove_type='wazuhdb'):
+    if remove_type not in ['wazuhdb','manage_agents']:
+        raise ValueError("Invalid type of agent removal: %s" % remove_type)
+
+    if agents_id:
+        for agent_id in agents_id:
+            if remove_type == 'manage_agents':
+                subprocess.call([f"{WAZUH_PATH}/bin/manage_agents", "-r", f"{agent_id}"], stdout=open(os.devnull, "w"),
+                                stderr=subprocess.STDOUT)
+            elif remove_type == 'wazuhdb':
+                result = query_wdb(f"global delete-agent {str(agent_id)}")
+
+
+def remove_all_agents(remove_type):
+    agent_ids_list = list_agents_ids()
+    remove_agents(agent_ids_list, remove_type)
```

Comment:
```suggestion
    remove_agents(list_agents_ids(), remove_type)
```
If it is possible, we could call the function `list_agents_ids()` inside `remove_agents` and the code would be simpler.

Reply:
Done d2419f50de93aa3c8c7ce0968c6abbe81321f316